Formula/libbpf.rb

```diff
@@ -0,0 +1,35 @@
+class Libbpf < Formula
+  desc "Automated upstream mirror for libbpf stand-alone build"
+  homepage "https://github.com/libbpf/libbpf"
+  url "https://github.com/libbpf/libbpf/archive/refs/tags/v0.8.1.tar.gz"
+  sha256 "7bda8187efc619d1eb20a1ba5ab949dd68d40dd44945310c91ac0f915fa4a42b"
+  license "BSD-2-Clause"
+
+  depends_on "linux-headers@5.16" => :test
+  depends_on "elfutils"
+  depends_on :linux
+  depends_on "pkg-config"
+  depends_on "zlib"
```

Comment:
```suggestion
  uses_from_macos "zlib"
```
Sorry to nitpick but It's probably better for us to use this just to maintain consistency with other formulae, even though this is Linux-only so it looks a bit weird to reference macOS.  `uses_from_macos` lines come after regular `depends_on` lines with a blank line separating them for readability.